pkg/arvo/sys/vane/clay.hoon

```diff
@@ -4932,6 +5025,37 @@
     |=  raf=raft-11
     ^-  raft-12
     raf(pud [pud.raf 0 0])
+  ::  +raft-12-to-13:
```

Comment:
@jtobin  Since this is likely to be released after the agents-in-clay work, I think it would make more sense to write the migration into the state for that branch.  Might want to retarget this at philip/agent-clay until that makes it into next/arvo, too.

Reply:
No sweat, can do.

Reply:
I've updated the base to philip/agent-clay; there's a conflict arising in the state migration, of course, but it looks to be easily resolvable on merge -- the base version numbers are to be preferred, and the handlers in +raft-12-to-13 probably should clear the flow/flue caches (or maybe wrapping the contents up in traps is safe for the time being).

Reply:
(I'll probably just merge in the branch and resolve it myself to save some hassle.)